routers/votes.router.js

```diff
@@ -33,7 +33,7 @@ router.post("/addVoteToRecord", (request, response) => {
   );
 });
 
-router.delete("/removeVotesFromContribution/:id", (request, response) => {
+router.delete("/votes/:id", (request, response) => {
```

Comment:
You don't need to put this path on every endpoint, you could just set it once in ./api.router.js and all these will get it.